plenum/server/node.py

```diff
@@ -3209,11 +3210,11 @@ def updateSeqNoMap(self, committedTxns, ledger_id):
             self.seqNoDB.addBatch((get_digest(txn), ledger_id, get_seq_no(txn))
                                   for txn in committedTxns)
 
-    def commitAndSendReplies(self, ledger_id, ppTime, reqs: List[Request],
+    def commitAndSendReplies(self, ledger_id, ppTime, reqs_keys,
```

Comment:
Please have a look at `executePoolTxnBatch` in `pool_manager.py`. I think it also needs to be modified accordingly.

Reply:
Done.